### PR TITLE
DOCS-11751-added the permissions information

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc [hunk 1]
```diff
@@ -5,12 +5,21 @@ include::_attributes.adoc[]
...
 :keywords: grant access, api, contracts
 
-After the user xref:exchange::to-request-access.adoc[requests access] to your API instance from Anypoint Exchange, as the Organization Administrator, you can manually approve or reject requests from API Manager if you have configured the SLA tier for the application for manual approvals.
+After users xref:exchange::to-request-access.adoc[request access] to your API instance from Anypoint Exchange, you can manually approve or reject requests from API Manager. 
+
+== Task Prerequisite
+
+To manually approve or reject requests, you must have:
+
+* The Organization Administrator or Manage Contracts permission
```
Comment (fedebrasburg) [inline modules/ROOT/pages/manage-client-apps-latest-task.adoc]: The `Environment administrator` permission also let the user approve contracts requests

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc [hunk 2]
```diff
@@ -5,12 +5,21 @@ include::_attributes.adoc[]
...
+* Configured the SLA tier for the application
```
Comment (fedebrasburg) [inline modules/ROOT/pages/manage-client-apps-latest-task.adoc:14]: this is like a prerequisite right? There wouldn't be nothing to approve/reject if there is no tier 😄

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-client-apps-latest-task.adoc]: Done! Thanks!

**Review by sarathecoubian — DISMISSED** [review]

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-client-apps-latest-task.adoc:14]: Yes, this is a part of the Prerequisite section, right?

Comment (fedebrasburg) [reply modules/ROOT/pages/manage-client-apps-latest-task.adoc:14]: oh yes, my bad, thanks!

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc [hunk 3]
```diff
@@ -11,8 +11,8 @@ After users xref:exchange::to-request-access.adoc[request access] to your API in
 
 To manually approve or reject requests, you must have:
 
-* The Organization Administrator or Manage Contracts permission
 * Configured the SLA tier for the application 
+* The Organization Administrator or Environment Administrator roles, or the Manage Contracts permission
```
Comment (fedebrasburg) [inline modules/ROOT/pages/manage-client-apps-latest-task.adoc]: `Environment administrator` is also a permission in Access Management, not a role 
image

**Review by sarathecoubian — APPROVED** [review]

**Review by fedebrasburg — APPROVED** [review]

**Pull request merged**